components/layouts/PageFooter/FooterSection.vue

```diff
@@ -2,20 +2,19 @@
   <section class="footer-section">
     <h2
       class="footer-section__title"
-      :class="{'footer-section__title_dark-mode': darkMode }"
+      :class="`footer-section__title_${mode}-mode`"
     >
       {{ title }}
     </h2>
     <nav :class="{ 'footer-section__icons-group': iconsOnly }">
       <AppLink
         v-for="element in elements"
         :key="element.url"
-        :class="{
-          'footer-section__icon-link': iconsOnly,
-          'footer-section__icon-link_dark-mode': darkMode,
-          'footer-section__link': !iconsOnly,
-          'footer-section__link_dark-mode': darkMode
-        }"
+        :class="
+          iconsOnly
+            ? `footer-section__icon-link_${mode}-mode`
```

Comment:
The element class is empty, that is why I removed it

Reply:
According to BEM, [a modifier cannot be used alone](https://en.bem.info/methodology/quick-start/#a-modifier-cant-be-used-alone). According to what I read in the BEM specification, this is for highlighting the modifier is "modifying" something and that something is the Element and a ward against future changes in the Element since the Element should factor-out the comon behavior.